source/operations/concepts/erasure-coding.rst

```diff
@@ -192,4 +182,4 @@ Whatever the cause, the result is the same - compromised data.
 
 MinIO’s optimized implementation of the :minio-git:`HighwayHash algorithm <highwayhash/blob/master/README.md>` ensures that it captures and heals corrupted objects on the fly. 
 Integrity is ensured from end to end by computing a hash on READ and verifying it on WRITE from the application, across the network, and to the memory or drive. 
```

Comment:
I'm having trouble parsing what exactly this is saying. Is it that MinIO computes the hash when data is read, and then verifies it as that data is written (which could follow multiple paths to the ultimate destination).